doc/Learn/Tutorials/Navigation/Advanced/HowTo-UsePanel.md

```diff
@@ -19,59 +19,79 @@ uid: Learn.Tutorials.Navigation.Advanced.Panel
 
 #### Navigating using a Panel
 
-```xml
-<Page x:Class="UsingPanelRegion.Views.MainPage"
-   xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
-   xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-   xmlns:local="using:UsingPanelRegion.Views"
-   xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
-   xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
-   mc:Ignorable="d"
-   xmlns:uen="using:Uno.Extensions.Navigation.UI"
-   xmlns:utu="using:Uno.Toolkit.UI"
-  Background="{ThemeResource MaterialBackgroundBrush}">
+    ```xml
```

Comment:
no improvement or arguably worse

Reply:
but keep the tab/space change

Reply:
> ![image](https://user-images.githubusercontent.com/2359550/221989286-8fce9705-713d-4a80-845e-0cab49c534c9.png)
> -- https://github.com/unoplatform/uno.extensions/blob/dev/lubl/fix-docs/doc/Learn/Tutorials/Navigation/Advanced/HowTo-UsePanel.md#navigating-using-a-panel

😬 dont 4-space indent unless within bullet-list-item, cause that is a syntax for language-less codeblock

Reply:
@Xiaoy312 Interesting. Didn't know that about the code block syntax. It should be addressed with the latest commit.